examples/liquidity_pool/src/main.sw

```diff
@@ -19,7 +19,9 @@ abi LiquidityPool {
     fn withdraw(recipient: Address);
 }
 
-const BASE_TOKEN = 0x9ae5b658754e096e4d681c548daf46354495a437cc61492599e33fc64dcdc30c;
+const BASE_TOKEN: AssetId = AssetId {
```

Comment:
Can we make this into a `from` and `into` function?

Reply:
I dont know what subid is used for that

Reply:
See prev reply for why i havent implemented a From for this struct

Reply:
Since this is a `b256` the `from` implementation here should be used. Especially since this is in the examples folder.